client/src/components/Event/Event.js

```diff
@@ -10,7 +10,7 @@ function Event() {
     const [isLoading, setLoading] = useState(true);
     useEffect(() => {
     axios
-      .get(`${process.env.REACT_APP_API || ''}/event/all`)
+      .get(`${process.env.REACT_APP_API || ''}/event`)
```

Comment:
This need not be changed. We want to display all the events on this page until #116 is completely solved.

Reply:
This will still request for all the events as the query option is not being passed to the API, referring to the new API structure in PR #192, in which `/event/all` and `/event` api is being merged. Correct me if I'm wrong